roles/upload-site/tasks/main.yml

```diff
@@ -84,18 +84,16 @@
     - "cc all"
     - "sqlq 'update system set status=\"1\" where name=\"stanford_dept\"'"
     - "-y en acsf"
-    - "ev 'acsf_openid_allow_local_user_logins();'"
-    - "-y rr"
+    - "rr"
   when: dept_site == "TRUE"
   notify: Clear site cache
 
 - name: Do post-database-restore tasks
   shell: "{{ drush_alias }} {{ item }}"
   with_items:
-    - "-y en acsf stanford_ssp paranoia"
-    - "ev 'acsf_openid_allow_local_user_logins();'"
+    - "-y en acsf acsf_helper stanford_ssp paranoia"
     - 'ev "_paranoia_remove_risky_permissions();"'
-    - "-y dis stanford_afs_quota"
+    - "-y dis stanford_afs_quota acsf_openid openid"
```

Comment:
I had to disable with `$disable_dependents = FALSE` in the installation scripts and I can't tell if drush does that.

https://github.com/SU-SWS/stanford_install_tasks/blob/7.x-1.x/Environment/ACSF/Install/EnableModules.php#L24
https://api.drupal.org/api/drupal/includes%21module.inc/function/module_disable/7.x
https://drushcommands.com/drush-7x/pm/pm-disable/

Also, we will want to uninstall these modules and not just disable.

Reply:
drush disables dependent modules. So we could just go with disabling `openid` here but I would like to be explicit.